source/simulator/simulator_access.cc

```diff
@@ -321,6 +321,19 @@ namespace aspect
 
 
 
+  template <int dim>
+  void
+  SimulatorAccess<dim>::compute_KXRCF_indicators(Vector<float> &KXRCF_indicators,
+                                                 const unsigned int field_index) const
+  {
+    const typename Simulator<dim>::AdvectionField advection_field =
+      (field_index == 0 ? Simulator<dim>::AdvectionField::temperature()
+       : Simulator<dim>::AdvectionField::composition(field_index-1));
+    simulator->compute_KXRCF_indicators(KXRCF_indicators, advection_field);
+  }
```

Comment:
This is not quite how we handle distinctions between temperature and composition in SimulatorAccess at the moment. Ideally we would be able to hand over `AdvectionField` arguments, but they are currently part of the Simulator class, which is a problem (see #5887). Can you model this function after `get_artificial_viscosity` and `get_artificial_viscosity_composition`, i.e. create two functions for temperature and composition? It is not ideal, but until #5887 is fixed I would rather have all functions in SimulatorAccess solve this in the same way.